### PR TITLE
GH-83893: Cross reference env. vars and -X command line options

### PR DESCRIPTION
**Pull request by furkanonder**
* Issue: gh-83893

Co-authored-by: Erlend E. Aasland

### CODE REVIEW

File: Doc/using/cmdline.rst [hunk 1]
```diff
@@ -495,16 +495,17 @@ Miscellaneous options
    Reserved for various implementation-specific options.  CPython currently
    defines the following possible values:
 
-   * ``-X faulthandler`` to enable :mod:`faulthandler`;
+   * ``-X faulthandler`` to enable :mod:`faulthandler`. See also
+   * :envvar:`PYTHONFAULTHANDLER`.
```
Comment (erlend-aasland) [inline Doc/using/cmdline.rst]:
```suggestion
   * ``-X faulthandler`` to enable :mod:`faulthandler`.
     See also :envvar:`PYTHONFAULTHANDLER`.
```

File: Doc/using/cmdline.rst [hunk 2]
```diff
@@ -495,16 +495,17 @@ Miscellaneous options
...
      memory blocks when the program finishes or after each statement in the
      interactive interpreter. This only works on :ref:`debug builds
      <debug-build>`.
    * ``-X tracemalloc`` to start tracing Python memory allocations using the
      :mod:`tracemalloc` module. By default, only the most recent frame is
      stored in a traceback of a trace. Use ``-X tracemalloc=NFRAME`` to start
-     tracing with a traceback limit of *NFRAME* frames. See the
-     :func:`tracemalloc.start` for more information.
+     :func:`tracemalloc.start` and :envvar:`PYTHONTRACEMALLOC` for more
+     information.
```
Comment (erlend-aasland) [inline Doc/using/cmdline.rst]:
```suggestion
     tracing with a traceback limit of *NFRAME* frames.
     See :func:`tracemalloc.start` and :envvar:`PYTHONTRACEMALLOC`
     for more information.
```

File: Doc/using/cmdline.rst [hunk 3]
```diff
@@ -861,16 +863,17 @@ conflict.
    Python memory allocations using the :mod:`tracemalloc` module. The value of
    the variable is the maximum number of frames stored in a traceback of a
    trace. For example, ``PYTHONTRACEMALLOC=1`` stores only the most recent
-   frame. See the :func:`tracemalloc.start` for more information.
+   frame. See the :func:`tracemalloc.start` for more information. This is
+   equivalent to setting the :option:`-X` ``tracemalloc`` option.
```
Comment (erlend-aasland) [inline Doc/using/cmdline.rst]:
```suggestion
   frame. See the :func:`tracemalloc.start` for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

File: Doc/using/cmdline.rst [hunk 4]
```diff
@@ -861,16 +863,17 @@ conflict.
...
    .. versionadded:: 3.4
 
 
 .. envvar:: PYTHONPROFILEIMPORTTIME
 
    If this environment variable is set to a non-empty string, Python will
-   show how long each import takes.  This is exactly equivalent to setting
-   ``-X importtime`` on the command line.
+   show how long each import takes.  This is equivalent to setting the
+   :option:`-X` ``importtime`` option.
```
Comment (erlend-aasland) [inline Doc/using/cmdline.rst]:
```suggestion
   show how long each import takes.
   This is equivalent to setting the :option:`-X` ``importtime`` option.
```

File: Doc/using/cmdline.rst (hunk 3 above)
Comment (CAM-Gerlach) [reply Doc/using/cmdline.rst]: Fix grammar issue (from the original, but on a modified line): either

```suggestion
   frame. See :func:`tracemalloc.start` for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

or

```suggestion
   frame. See the :func:`tracemalloc.start` function for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

**Review by CAM-Gerlach — COMMENTED** [review]
**Standard reminder: You can directly apply all the suggestions you want in one go by going to [`Files changed`](https://github.com/python/cpython/pull/103414/files) -> Clicking <kbd>Add to batch</kbd> on each suggestion -> When done, clicking <kbd>Commit</kbd>**

One suggestion, aside from those @erlend-aasland already made.

Just to note, the docs build error is unrelated and will be fixed by #103421

**Review by CAM-Gerlach — APPROVED** [review]
LGTM from me; thanks @furkanonder

**Review by erlend-aasland — COMMENTED** [review]
LGTM, and thanks for picking this up, Furkan!

**Review by erlend-aasland — APPROVED** [review]

File: Doc/using/cmdline.rst [hunk 5]
```diff
@@ -861,16 +864,17 @@ conflict.
    Python memory allocations using the :mod:`tracemalloc` module. The value of
    the variable is the maximum number of frames stored in a traceback of a
    trace. For example, ``PYTHONTRACEMALLOC=1`` stores only the most recent
-   frame. See the :func:`tracemalloc.start` for more information.
+   frame. See the :func:`tracemalloc.start` function for more information.
+   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```
Comment (erlend-aasland) [inline Doc/using/cmdline.rst]: I like to sneak in SemBr for every touched line, so let's do it here as well. #sneaksembrification

```suggestion
   frame.
   See the :func:`tracemalloc.start` function for more information.
   This is equivalent to setting the :option:`-X` ``tracemalloc`` option.
```

**Pull request merged**